.chloggen/kubeletAKS.yaml

```diff
@@ -0,0 +1,12 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+# The name of the component, or a single word describing the area of concern, (e.g. agent, clusterReceiver, gateway, operator, chart, other)
+component: agent
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Configure AKS KubeletStats receiver to use the appropriate CA file
```

Comment:
Should we add a link to the new secton in the advanced docs?

Reply:
yes, I will do that